### ISSUE TITLE
Dev UI: Links in To get started broken

### ISSUE THREAD

**Issue by sitapriyamoorthi** [FredHutch/shiny-cromwell#185]
The links in the homepage dont work!
To get Started links only. The ones above them work!

**Comment by sckott** [comment]
Thanks @sitapriyamoorthi but this is hard to follow. They do work for me but it does take a while

**Comment by sckott** [comment]
I'll investigate

**Comment by sitapriyamoorthi** [comment]
I just checked and looks like it works on Dev now! I am going to close this issue!
